pkg/dynamicinstrumentation/diconfig/binary_inspection.go

```diff
@@ -21,23 +21,26 @@ import (
 // inspectGoBinaries goes through each service and populates information about the binary
 // and the relevant parameters, and their types
 // configEvent maps service names to info about the service and their configurations
-func inspectGoBinaries(configEvent ditypes.DIProcs) error {
+func inspectGoBinaries(configEvent ditypes.DIProcs) (map[ditypes.PID]bool, error) {
```

Comment:
maybe comment on the returned map, and also on when an error is returned?

Reply:
Done in 90c911b2c371aed58f206304eb575a2174cf3c3c.